src/components/dashboard/dashboardItem/AppItem.js

```diff
@@ -28,7 +28,7 @@ class AppItem extends ItemBase {
 
     static create(props) {
         const item = new AppItem(props);
-        const { showErrorAnnouncer } = props;
+        const { showErrorAnnouncer, setDetailsApp } = props;
         const { t } = useTranslation(["dashboard", "apps"]);
```

Comment:
Since `getFavoriteActionKey` was removed, then I think this can now be reverted:
```suggestion
        const { t } = useTranslation("dashboard");
```